### ISSUE TITLE
BUG REPORT BY GM-PRO

### ISSUE THREAD

**Issue by GM-PRO** [CaNNiC2/rbe-zombie-meat-4.3#2]
FIX THE BUG WHEN SO MANY PEOPLE HIT ONE ZOMBIE SERVER CRASHED

**Comment by RBEsLayeR** [comment]
Does it happens always ?
And does server goes completly off or it just FPS drop for some seconds ?

**Comment by GM-PRO** [comment]
> Does it happens always ?
> And does server goes completly off or it just FPS drop for some seconds ?

Yes when so many players kill one zombie server fully crashed I saw somany times it's happen

**Comment by RBEsLayeR** [comment]
Yes , because Xash3D engine kinda sucks 🤨

**Comment by KiLiSAMEYT** [comment]
Lol :p
